src/Atmos/Model/moisture.jl

```diff
@@ -76,25 +78,24 @@ function gradvariables!(::EquilMoist, transform::Vars, state::Vars, aux::Vars, t
 end
 
 
-function diffusive!(::EquilMoist, diffusive::Vars, ∇transform::Grad, state::Vars, aux::Vars, t::Real, ν::Union{Real,AbstractMatrix})
+function diffusive!(m::EquilMoist, diffusive::Vars, ∇transform::Grad, state::Vars, aux::Vars, t::Real, ν::Union{Real,AbstractMatrix})
   # turbulent Prandtl number
   diag_ν = ν isa Real ? ν : diag(ν) # either a scalar or vector
   D_q_vap = D_q_liq = D_q_ice = D_q_tot = diag_ν / Prandtl_t # either a scalar or vector
-  
+
   # diffusive flux of q_tot
-  # FIXME 
+  # FIXME
   ρd_q_vap = state.ρ * (-D_q_vap) .* ∇transform.moisture.q_vap # a vector
   ρd_q_liq = state.ρ * (-D_q_liq) .* ∇transform.moisture.q_liq # a vector
   ρd_q_ice = state.ρ * (-D_q_ice) .* ∇transform.moisture.q_ice # a vector
```

Comment:
We don't need the separate vapor, liquid, ice fluxes. Just compute total flux from gradient of total specific humidity. This will be more stable (less cancelation, e.g., between vapor and condensate at cloud boundaries).